### PR TITLE
Dynamic fee calculation 1/2

### PR DESCRIPTION
**Pull request by saiy2k**
Few Notes:
* newBoltzApi() iniaitlization url needs to be done proper.
* `claimFeesEstimate` and `lockupFees` values needs to be verified in `boltz-dart` if they are right.

### CODE REVIEW

File: lib/swap/bloc/swap_cubit.dart [hunk 1]
```diff
@@ -185,7 +197,7 @@ class SwapCubit extends Cubit<SwapState> {
     }
 
     final updatedSwap = swap!.copyWith(
-      boltzFees: (fees.btcSubmarine.boltzFeesRate * amount) as int,
+      boltzFees: (fees.btcSubmarine.boltzFeesRate * amount / 100) as int,
```
Comment (i5hi) [inline lib/swap/bloc/swap_cubit.dart]: we are using submarine here. should be reverse.

File: lib/receive/receive_page.dart [hunk 2]
```diff
@@ -59,7 +59,7 @@ class _ReceivePageState extends State<ReceivePage> {
       walletSensitiveRepository: locator<WalletSensitiveStorageRepository>(),
       swapBoltz: locator<SwapBoltz>(),
       walletTx: locator<WalletTx>(),
-    );
+    )..fetchFees(context.read<NetworkCubit>().state.testnet);
```
Comment (i5hi) [inline lib/receive/receive_page.dart:62]: why are we using testnet here? what if its mainnet?

Comment (saiy2k) [reply lib/receive/receive_page.dart:62]: So `context.read<NetworkCubit>().state.testnet` is a bool. It returns true/false, based on current network selection in app.

Comment (i5hi) [reply lib/receive/receive_page.dart:62]: ah okay. thanks!

**Pull request merged**